NewsHSE/Modules/News/Views/NewsTableManager.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  NewsTableManager.swift
+//  NewsHSE
+//
+//  Created by Darya Zhitova on 20.11.2023.
+//
+
+import UIKit
+
+protocol NewsTableManagerDelegate {
+    func didSelectRow(_ beerModel: NewsDetailDataFlow.Presentation.ViewModel)
+}
+
+final class NewsTableManager: NSObject {
+    var delegate: NewsTableManagerDelegate?
```

Comment:
Это ссылка на контроллер. Попадаем в цикл сильных ссылок.
```suggestion
    weak var delegate: NewsTableManagerDelegate?
```

Reply:
Видимо не тому делегату написал этот коммент.)
Все равно надо ослабить, но тут менеджер держит ссылку не на котроллер, а на вьюху через делегата. Вьюха держит ссылку на менеджер через свойство tableManager.  Снова цикл сильных ссылок, поэтому ослабляем ссылку на делегат.

Reply:
хорошо, спасибо за объяснение)